client/driver/docker.go

```diff
@@ -1000,67 +1009,99 @@ func (d *DockerDriver) cleanupImage(imageID string) error {
 	return nil
 }
 
+// dockerHealthCheckClient creates a single *docker.Client with a timeout of
+// one minute, which will be used when performing Docker health checks.
+func (d *DockerDriver) dockerHealthCheckClient() (*docker.Client, error) {
+	createClientsLock.Lock()
+	defer createClientsLock.Unlock()
+
+	if healthCheckClient != nil {
+		return healthCheckClient, nil
+	}
+
+	newHealthCheckClient, err := d.newDockerClient(dockerHealthCheckTimeout)
```

Comment:
May want to use the same `var err error` trick and assign to healthCheckClient directly.